hearth/sig/lib/hearth/interfaces.rbs

```diff
@@ -84,4 +84,10 @@ module Hearth
   interface _WritableIO
     def write: (*_ToS) -> Integer
   end
+
+  interface _ReadableIO
+    def read: (*_ToS) -> String
+    def readpartial: (Integer, ?String) -> String
+    def size: () -> Integer
```

Comment:
I think this might not always have size?

Reply:
Is there a way to put this as optional?